uint8 color
int32 h_min
int32 h_max
int32 s_min
int32 s_max
int32 v_min
int32 v_max
